### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by sigtot** [graphql-python/graphene-django#1205] [General]
Hey!

I'm wondering if a new release is planned any time soon? Last release was in December 2020 and since then many new additions have been put out. I know I'm eagerly waiting for some of the features added to the v2 branch, such as the ability to specify optional fields in DRF mutations (#1080).

**Comment by TurnrDev** [comment]
I too am wondering this, I seems this library is still in active development but 3.0 is still beta. Is there anything I can do to help development?
